integration_tests/test_doc.py

```
@@ -0,0 +1,230 @@
+    w3: AsyncWeb3 = mantra.async_w3
+    sender = grantor()
+    target = _editor1()
+    checksum = ""
+
+    tx = PRECOMPILE.fns.grantRole(REGISTRY_ID, checksum, target.address, Role.EDITOR)
+
+    if should_succeed:
+        receipt = await tx.transact(w3, sender, to=DOCUMENT)
+        assert receipt.status == 1, "GrantRole transaction failed"
+    else:
+        with pytest.raises(Exception):
```

Ruff (0.14.8)

Do not assert blind exception: `Exception`

(B017)